Support for .NET 6.0 & SkiaSharp Renderer

driekus77 opened the pull request:
- HtmlRenderer.PdfSharp for GDI based rendering: Tested with the Winforms App.
- HtmlRenderer.SkiaSharp for Rendering using SkiaSharp: Work in progress!

Using PdfSharpCore in stead of PdfSharp.

Is far a I can see the demo's are working. (Except the external images because of death links).

Commit:
Introducing net6.0 port of HtmlRenderer

- HtmlRenderer.PdfSharp for GDI based rendering: Tested with the Winforms App.
- HtmlRenderer.SkiaSharp for Rendering using SkiaSharp: Work in progress!

mstancombe commented:
@ArthurHub what is the process for getting this merged in?  Do you want a hand with reviewing or such?  I'd be happy to help out.

mstancombe committed:
Merge pull request #2 from driekus77/features/net60

Introducing net6.0 port of HtmlRenderer

mstancombe committed:
Adding demo app to compare, Modify PdfSharp to use Async invocation, WIP on Skia.

mstancombe committed:
Updated to .net 7, lots more work getting Skia font sizing more accurately matching PDFSharp, Implemented lines/rectangles in Skia.

mstancombe committed:
Updated remaining projects to net7, some tweaks for winforms sample.

driekus77 committed:
Update README.md

Test github

mstancombe committed:
Update skia dependency

mstancombe committed:
Adding build pipeline for the net6 branch to push Core and Skia.

mstancombe committed:
Add project URL

mstancombe committed:
Set build scripts to net 7

mstancombe committed:
Removed non cross plat rendering targets (Win/WPF) from default builds.  These might just be removed permenantly in future.

mstancombe committed:
Fix case issue

mstancombe committed:
Fix issue where core not published

mstancombe committed:
Image Fixes
-Implemented image loading using HttpClient
-Added demo resources images to console program
-Moved sample test png expected location out of root c: folder.  Ideally this should be something less platform specific, but haven't crossed that bridge yet.

mstancombe committed:
Add SVG Support

mstancombe committed:
Fix issue where images not waiting for load.

mstancombe committed:
Add basic page-break 'always' support, remove default page breaking on printed h1 element.

Commit:
Swap skia svg package

mstancombe committed:
Merge pull request #3 from albyoo/features/net60

Swap Skia SVG NuGet Package

Commit:
Rename corner radius to border radius

Commit:
Update demo output to use date format yyyyMMdd

Commit:
Implement skia border radius

mstancombe committed:
Merge pull request #4 from albyoo/features/border-radius

Rename corner-radius to border-radius & Implement Skia border-radius.

Commit:
Update graphics adapter to use correct calculated height

Commit:
Update skia sharp adapter to use font manager default singleton

mstancombe committed:
Merge pull request #6 from albyoo/features/font-sizing

Update font size caluclation

driekus77 committed:
Merge pull request #1 from mstancombe/features/net60

Features/net60

eXpl0it3r commented:
Closing in favor of #234

Will look into the SkiaRenderer approach
